js/tabs/js/lumx.tabs_directive.js

```diff
@@ -75,6 +76,184 @@ angular.module('lumx.tabs', [])
             }
         };
 
+        function isPaginationActive() 
+        {
+            var tabsWidth = links.outerWidth();
+            var tabsVisibleWidth = links.parent('.tabs').outerWidth();
```

Comment:
The `links.parent('.tabs')` is used multiple times. It would be better to store it in a local var.